src/main/java/frc/robot/commands/swerve/TurnToAngleVision.java

```diff
@@ -42,7 +44,17 @@ public void execute() {
         double rightY = RobotContainer.constrainJoystick(RobotContainer.rightJoystick.getY());
 
         if (latestResult == null) {
-            omega = 0;
+            if(DriverStation.isFMSAttached()) {
+                Pose2d robotPose = RobotContainer.driveTrain.getEstPose2D();
+                double facingInRadians = RobotContainer.navX.getRotation2d().getRadians();
+                double x = robotPose.getX();
+                double y = robotPose.getY();
+                double targetRadians = Math.atan2(Parameters.vision.GOAL_X - y, Parameters.vision.GOAL_Y - x);
+                double closestAngle = (targetRadians - facingInRadians + Math.toRadians(540)) % Math.toRadians(360) - Math.toRadians(180);
+                omega = Math.signum(closestAngle);
```

Comment:
Invert this angle, otherwise this is some cool stuff. No idea how it works, but it does.